source/js/add-book.js

```diff
@@ -1,63 +1,83 @@
 /* eslint-disable linebreak-style */
+/* eslint-disable class-methods-use-this */
 /* eslint-disable object-shorthand */
```

Comment:
## [OPTIONAL]
* I strongly suggest that you try to fix the linter errors rather than disabling it as it would help you become a better programmer